packages/patterns-react/src/patterns/ListSection/ListGroup.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import List from './List';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * List Group Component
+ *
+ * @param {object} props props object
+ * @param {string} props.title List Group title
+ * @param {string} props.lists link Group lists object
+ * @returns {object} JSX Object
+ */
+const ListGroup = ({ listGroup: { title, lists } }) => {
+  return (
+    <div
+      data-autoid={`${stablePrefix}--listgroup`}
+      className={`${prefix}--listgroup ${prefix}--listgroup__col`}>
+      <div className={`${prefix}--listgroup__title`}>{title}</div>
+      <div className={`${prefix}--listgroup__list`}>{renderList(lists)}</div>
+    </div>
+  );
+};
+
+/**
+ * Render List Component
+ *
+ * @param {object} listItems listItems Object
+ * @returns {object} JSX Object
+ */
+const renderList = listItems => {
```

Comment:
Minor thing, if this is a private method, should have an underscore in the front of the name (`_renderList`), and mark as `@private` in jsdocs.

Reply:
will do

Reply:
marked renderList function as private by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/573/commits/6b28d40fcd3278749c38a30d336f80c5f19b5a11)